actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsBackgroundActor.java

```diff
@@ -71,7 +74,9 @@ public void courseProgressMetricsData(Request actorMessage) {
     List<Map<String, Object>> responseList =
         (List<Map<String, Object>>) response.get(JsonKey.RESPONSE);
     if (responseList.isEmpty()) {
-      ProjectLogger.log("Invalid data");
+      ProjectLogger.log(
+          "CourseMetricsBackgroundActor:courseProgressMetricsData Invalid data",
```

Comment:
From:
"CourseMetricsBackgroundActor:courseProgressMetricsData Invalid data"

To:
"CourseMetricsBackgroundActor:courseProgressMetricsData: requestId not found."